sdk/eventhub/event-hubs/src/pumpManager.ts

```diff
@@ -25,8 +25,8 @@ export interface PumpManager {
    * @ignore
    */
   createPump(
+    startPosition: EventPosition,
```

Comment:
Slightly curious why you changed the ordering :)  The TSDoc comments are no longer in sync!

Reply:
My thought was that the startPosition was the "front and center" parameter so it should be first. 

I could be wrong....

Reply:
Btw, pushed a change that fixes the typedoc.